third_party/move/move-compiler-v2/src/pipeline/livevar_analysis_processor.rs

```diff
@@ -218,17 +219,12 @@ impl LiveVarAnalysisProcessor {
 // Dataflow Analysis
 
 /// State of the livevar analysis,
-#[derive(Debug, Clone, Eq, PartialEq, PartialOrd)]
+#[derive(AbstractDomain, Debug, Clone, Eq, PartialEq, PartialOrd)]
 struct LiveVarState {
+    #[join]
```

Comment:
I would find it more natural to have the default be to attempt a join and use `#[ignore]` to skip the field. That is also how Serde and Clap are dowing it.

Reply:
+1